src/client.ts

```diff
@@ -163,6 +177,13 @@ export const createStorefrontClient = (options: StorefrontClientOptions) => {
         return;
       }
 
+      if (cartTokenStorageType === 'cookie') {
+        return Cookies.remove(
```

Comment:
You also need to set the `storedCartToken` to `null`. So this should be something like:

```
Cookies.remove(
	`${storageKeyPrefix}${STOREFRONT_CART_TOKEN_STORAGE_NAME}`,
	cartTokenCookieOptions,
);
storedCartToken = null;
return;
```